src/main/java/org/snomed/snowstorm/core/data/domain/Annotation.java

```diff
@@ -0,0 +1,137 @@
+package org.snomed.snowstorm.core.data.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonView;
+import io.micrometer.core.instrument.util.StringUtils;
+import org.snomed.snowstorm.core.pojo.TermLangPojo;
+import org.snomed.snowstorm.rest.View;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@JsonView(value = View.Component.class)
+@JsonIgnoreProperties({"memberId", "additionalFields", "effectiveTimeI", "deleted", "changed"})
+public class Annotation extends ReferenceSetMember implements AnnotationView {
+
+	private String annotationId;
+	private String annotationTypeId;
+	private ConceptMini annotationType;
+	private String annotationValue;
+	private String annotationLanguage;
+
+	@Override
+	public String getAnnotationId() {
+		return annotationId;
+	}
+
+	public void setAnnotationId(String annotationId) {
+		this.annotationId = annotationId;
+	}
+
+	@Override
+	public String getAnnotationTypeId() {
+		return annotationTypeId;
+	}
+
+	public void setAnnotationTypeId(String annotationTypeId) {
+		this.annotationTypeId = annotationTypeId;
+	}
+
+	@JsonIgnore
+	public ConceptMini getAnnotationType() {
+		return annotationType;
+	}
+
+	@Override
+	@JsonView(value = View.Component.class)
+	public TermLangPojo getAnnotationTypePt() {
+		return annotationType != null ? annotationType.getPt() : null;
+	}
+
+	public void setAnnotationType(ConceptMini annotationType) {
+		this.annotationType = annotationType;
+	}
+
+	@Override
+	public String getAnnotationValue() {
+		return annotationValue;
+	}
+
+	public void setAnnotationValue(String annotationValue) {
+		this.annotationValue = annotationValue;
+	}
+
+	@Override
+	public String getAnnotationLanguage() {
+		return annotationLanguage;
+	}
+
+	public void setAnnotationLanguage(String annotationLanguage) {
+		this.annotationLanguage = annotationLanguage;
+	}
+
+	public Annotation fromRefsetMember(ReferenceSetMember fromMember) {
+		setAnnotationId(fromMember.getMemberId());
+		setMemberId(fromMember.getMemberId());
+		setRefsetId(fromMember.getRefsetId());
+		setModuleId(fromMember.getModuleId());
+		setActive(fromMember.isActive());
+		setEffectiveTimeI(fromMember.getEffectiveTimeI());
+		setReferencedComponentId(fromMember.getReferencedComponentId());
+		setReleased(fromMember.isReleased());
+		setAnnotationTypeId(fromMember.getAdditionalField(AnnotationFields.ANNOTATION_TYPE_ID));
+
+		String annotationValue = fromMember.getAdditionalField(AnnotationFields.ANNOTATION_VALUE);
+		setAnnotationValue(annotationValue);
+		if (StringUtils.isNotEmpty(annotationValue) && annotationValue.startsWith("@")) {
+			setAnnotationLanguage(annotationValue.substring(1, annotationValue.indexOf("\"")));
```

Comment:
@QuyenLy87, when creating a Concept with an annotation, the `annotationLanguage` property is not returned.

Reply:
Hi @dmcgihtsdo, the annotation value should be @en"value", then you can retrieve the annotationLanguage